internal/settings/settings.go

```diff
@@ -26,4 +25,7 @@ func Initialize() {
 	// Modes
 	viper.SetDefault("Async", false)
 	viper.SetDefault("Verbose", false)
+
+	// Constants
+	viper.SetDefault("DockerAPIVersion", "1.39")
```

Comment:
I guess we can override this from setting.yaml or Setting en ENV as `dunner_DockerAPIVersion`?

Reply:
Yes, it will be automatically updated by `viper.AutomaticEnv()`.

Reply:
can we document all these in a Wiki?

Reply:
Yeah. I will work upon the cookbook and documentation, including Wiki soon after completing some of the milestones of `v1.0`.